### PR TITLE
ecscontainermetrics: update otconfig, expected metrics, and Validator logic

### PR DESCRIPTION
**Pull request by hossain-rayhan**
This change updates-
1. OT Config for ECS Container Metrics Receiver
2. Expected metrics for ECS Container Metrics Receiver
3. Validation logic for empty dimensions

Note: Once the dimensions/label settings feature is available in the exporter, we will create another PR to update our otconfig and expected metrics file.

### CODE REVIEW

**Comment by wyTrivail** [comment]
thanks! could you fix the build?

File: validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache [hunk 1]
```diff
@@ -1,22 +1,32 @@
+-
+  metricName: ecs.task.memory.reserved_{{testingId}}
+  namespace: {{metricNamespace}}
+  dimensions:
```
Comment (wyTrivail) [inline validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache:4]: let's wait until you have the real dimensions, then we can merge

**Review by wyTrivail — CHANGES_REQUESTED** [review]
looks good to me overall, let's wait until you have the real dimensions, then we can merge it, thanks!

File: validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache (hunk 1 above)
Comment (hossain-rayhan) [reply validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache:4]: We are planning release with empty dimensions. Will create a new PR once the label/dimensions settings feature is available.

**Pull request merged**